src/util/validators/common-rules.ts

```diff
@@ -0,0 +1,8 @@
+export const EMAIL_VALIDATION_RULES = {
+  regexp: /^(?=.{1,254}$)(?=.{1,64}@)[-!#$%&'*+/0-9=?A-Z^_`a-z{|}~]+(\.[-!#$%&'*+/0-9=?A-Z^_`a-z{|}~]+)*@[A-Za-z0-9]([A-Za-z0-9-]{0,61}[A-Za-z0-9])?(\.[A-Za-z0-9]([A-Za-z0-9-]{0,61}[A-Za-z0-9])?)*$/
+};
```

Comment:
Add reference to the source.
@AHGIJMKLKKZNPJKQR Does it match with rules in the backend?

Reply:
`/^[a-zA-Z0-9.!#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/` this is the regex used by the backend under the hood.

Reply:
used @AHGIJMKLKKZNPJKQR  regex and left source in comment